### PR TITLE
Easier setup on Swift 3

### PR DESCRIPTION
**Pull request by jaimeagudo**
Fix podfile repo branch
Add theme usage example for Swift

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -729,6 +729,14 @@ To set a global theme, you can do the following in your app delegate:
 [Chameleon setGlobalThemeUsingPrimaryColor:(UIColor *)color withContentStyle:(UIContentStyle)contentStyle];
 ```
 
+######Swift
+``` swift
+Chameleon.setGlobalThemeUsingPrimaryColor(.flatPlum(),
```
Comment (bre7) [inline README.md]: .flatPlum instead of .flatPlum()

File: README.md [hunk 2]
```diff
@@ -729,6 +729,14 @@ To set a global theme, you can do the following in your app delegate:
...
+                                                  withSecondaryColor: .flatBlue(),
```
Comment (jaimeagudo) [inline README.md]: Nope, Swift 3.0 compiler on Xcode 3.0 says I am right :)

File: README.md (hunk 1 above)
Comment (jaimeagudo) [inline README.md]: Nope, Swift 3.0 compiler on Xcode 3.0 says I am right :)

Comment (bre7) [inline README.md]: Are you using HEAD ?

Comment (jaimeagudo) [inline README.md]: `swift3` branch ofc :) , the compiler moans for the lack of `()`

Comment (bre7) [inline README.md]: Update to 8.1. I've just cloned the repo and it works without the () (as expected)

Comment (bre7) [inline README.md]: screen shot 2016-11-23 at 11 52 42

Comment (jaimeagudo) [inline README.md]: I experience a weird bug where I cannot upgrade to 8.1 so I will believe you :)

File: README.md [hunk 3]
```diff
@@ -16,7 +16,7 @@
 
 To use the Swift 3 version, add this to your Podfile (until 2.2 or higher is released):
 ```ruby
-pod 'ChameleonFramework/Swift', :git => 'https://github.com/ViccAlexander/Chameleon.git'
+    pod 'ChameleonFramework/Swift', :git => 'https://github.com/ViccAlexander/Chameleon.git', :branch => 'swift3'
```
Comment (bre7) [inline README.md]: Master is swift3 compatible. `swift3` branch is an old one, I'll delete it later.

Comment (bre7) [inline README.md]: swift3 branch deleted. All of the changes were already included in master.

Comment (jaimeagudo) [inline README.md]: my .2$: a working example is always better to get new users than having to figure out these backs & forths and many people are on Xcode 8.0.

**Pull request closed without merging**